examples/bert_hf_cpu_aml/bert_config.json

```diff
@@ -1,11 +1,17 @@
 {
     "verbose": true,
+    "azureml_client": {
+        "subscription_id": "<place_holder>",
+        "resource_group": "<place_holder>",
+        "workspace_name": "<place_holder>"
+    },
     "input_model":{
         "type": "PyTorchModel",
         "config": {
             "name": "bert-hf",
             "version": "3",
             "model_storage_kind": "azureml",
+            "aml_storage_name": "bert_hf",
```

Comment:
How about move aml_client info from engine entry to input model's property, Like:
![image](https://github.com/microsoft/Olive/assets/13343117/b931bf27-f2d1-47c6-8c09-3b23dc356e76)